packages/celer-web-app/src/ui/frames/Home.tsx

```diff
@@ -8,9 +8,6 @@ import { EmptyObject } from "data/util";
 function populateRecentPages() {
 	const recentPages = loadRecentPages();
 	return recentPages.map((url) => {
-		if (!url) {
```

Comment:
Using `filter` will be cleaner 
```
return recentPages.filter(Boolean).map(url=> { ..... });
```
more details on why this works: https://michaeluloth.com/filter-boolean

Reply:
My bad.. didn't see this is a removal.
I think you should still add the filter